src/flow/node-inner.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styled, { css } from 'styled-components'
+
+
+const humanTime = (seconds) => {
```

Comment:
should just make humanTime a helper function thats imported here and in dagre/index.js